src/instruments/src/RMP/Components/StandbyFrequency.tsx

```diff
@@ -103,29 +103,40 @@ export const StandbyFrequency = (props: Props) => {
     const spacing = usePersistentProperty('RMP_VHF_SPACING_25KHZ', '0')[0] === '0' ? 8.33 : 25;
     // Handle outer knob turned.
     const outerKnobUpdateCallback: UpdateValueCallback = useCallback((offset) => {
-        const frequency = Math.round(props.value / 1000);
-        const integer = Math.floor(frequency / 1000);
-        const decimal = frequency % 1000;
-        // @todo determine min/max depending on mode.
-        const maxInteger = decimal > 975 ? 135 : 136;
-        const newInteger = Utils.Clamp(integer + offset, 118, maxInteger);
-        props.setValue((newInteger * 1000 + decimal) * 1000);
+        if (props.value !== 0) {
+            const frequency = Math.round(props.value / 1000);
+            const integer = Math.floor(frequency / 1000);
+            const decimal = frequency % 1000;
+            // @todo determine min/max depending on mode.
+            const maxInteger = decimal > 975 ? 135 : 136;
+            const newInteger = Utils.Clamp(integer + offset, 118, maxInteger);
+            props.setValue((newInteger * 1000 + decimal) * 1000);
+        } else {
+            props.setValue(0);
+        }
     }, [props.value]);
 
     // Handle inner knob turned.
     const innerKnobUpdateCallback: UpdateValueCallback = useCallback((offset) => {
         const frequency = Math.round(props.value / 1000);
-        if (Math.sign(offset) === 1 && frequency === 136975) {
-            return;
+        // if (Math.sign(offset) === 1 && frequency === 136975) {
+        //     return;
+        // }
```

Comment:
These lines can be deleted.

Because it prevents the decimal value to go beyond 975 if the integer is 136. In real life, there is no such thing. The outer knob is blocked at 136 but the inner one can go to 0. There is a comment line 132.